completions/7z

```diff
@@ -6,7 +6,7 @@ _7z()
     _init_completion -n = || return
 
     if ((cword == 1)); then
-        COMPREPLY=($(compgen -W 'a b d e l t u x' -- "$cur"))
+        COMPREPLY=($(compgen -W 'a b d e h i l rn t u x' -- "$cur"))
```

Comment:
Should we (not) make `rn` a `mode=w` word a few lines below?

Reply:
Yes indeed.  Good call.  Fix pushed.